presto-product-tests/conf/presto/etc/singlenode-ldap.properties

```diff
@@ -0,0 +1,29 @@
+#
+# WARNING
+# ^^^^^^^
+# This configuration file is for development only and should NOT be used be
+# used in production. For example configuration, see the Presto documentation.
+#
+
+node.id=will-be-overwritten
+node.environment=test
+
+coordinator=true
+node-scheduler.include-coordinator=true
+http-server.http.port=8080
+query.max-memory=1GB
+query.max-memory-per-node=512MB
+discovery-server.enabled=true
+discovery.uri=http://presto-master:8080
+
+# LDAP specific properties
+# https will have to enabled for ldap authentication
+http-server.https.port=8443
+http-server.https.enabled=true
+http-server.https.keystore.path=/etc/ldap/coordinator.jks
+http-server.https.keystore.key=testldap
+authentication.ldap.enabled=true
+authentication.ldap.url=ldaps://ad-test.presto.testldap.com:636
+authentication.ldap.ad-domain=presto.testldap.com
```

Comment:
ditto?

P.S.: Take my appologies for the dumb question in advance. I have no idea how LDAP works internally.